plugins/net_plugin/net_plugin.cpp

```diff
@@ -3673,10 +3673,10 @@ namespace eosio {
       }
    }
 
-   void connection::handle_message( const chain::hs_message& msg ) {
-      peer_dlog(this, "received hs: ${msg}", ("msg", msg));
+   void connection::handle_message( const chain::hs_vote_message& msg ) {
+      peer_dlog(this, "received vote: ${msg}", ("msg", msg));
       controller& cc = my_impl->chain_plug->chain();
-      cc.notify_hs_message(connection_id, msg);
+      cc.process_vote_message(msg);
    }
```

Comment:
Need to also update `bcast_hs_message` to broadcast `hs_vote_message` instead of `hs_message`. Go ahead and remove `hs_message` so nothing else is missed.